_rules/element-lang-valid-de46e4.md

```diff
@@ -73,33 +77,33 @@ The `lang` attribute has a value that is not empty ("") and has a valid primary
 </html>
 ```
 
-### Failed
-
-#### Failed Example 1
+#### Passed Example 3
 
-The `lang` attribute value is not a valid primary language subtag.
+The `lang` attribute value has a valid primary language subtag, but a syntactically invalid region subtag.
 
 ```html
 <html>
 	<body>
-		<article lang="dutch"></article>
+		<p lang="en-US-GB"></p>
```

Comment:
This is right - BCP 47 has an algorithm for determining best matching language in 'Matching of Language Tags' at the end of BCP 47:

> In the lookup scheme, the language range is progressively truncated
   from the end until a matching language tag is located

So lookup order is `en-US-GB` to `en-US` to `en`